R/geom_phylopic.R

```diff
@@ -12,10 +12,15 @@
 #'   specifies the height of the silhouettes in the units of the y axis. The
 #'   aspect ratio of the silhouettes will always be maintained.
 #'
-#'   The `alpha` and `color` aesthetics can be used to change the transparency
-#'   and color of the silhouettes, respectively. If "original" is specified for
-#'   the `color` aesthetic, the original color of the silhouette will be used
-#'   (usually the same as "black").
+#'   The `color` (default: "transparent"), `fill` (default: "black"), and
+#'   `alpha` (default: 1) aesthetics can be used to change the outline color,
+#'   fill color, and transparency (outline and fill) of the silhouettes,
+#'   respectively. If only `color` is specified, the outline and fill color will
```

Comment:
This is the preferred way of doing it. However, it does not behave this way in add_phylopic_base. It's probably important to standardise this?

```
plot(1, 1, type = "n", main = "A cat")
add_phylopic_base(uuid = "23cd6aa4-9587-4a2e-8e26-de42885004c9",
                  color = "red",
                  x = 1, y = 1, ysize = 0.4)
```